storage/src/main/java/org/apache/uniffle/storage/request/CreateShuffleReadHandlerRequest.java

```diff
@@ -41,6 +41,7 @@ public class CreateShuffleReadHandlerRequest {
   private List<ShuffleServerInfo> shuffleServerInfoList;
   private Roaring64NavigableMap expectBlockIds;
   private Roaring64NavigableMap processBlockIds;
+  private int maxHanderFailTimes;
```

Comment:
typo `maxHanderFailTimes` -> `maxHandlerFailTimes`, or rename it to maxHandlerRetries?

Reply:
I think `maxHandlerFailTimes` is better

Reply:
Done

Reply:
ok